posts/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+#from users.models import User
+
+
+class Post(models.Model):
+	post_text  = models.CharField(max_length = 5000)
+	post_title = models.CharField(max_length = 100)
+	pub_date= models.DateTimeField('date field')
+#	post_creator = models.ForeignKey(User , on_delete= models.CASCADE)
```

Comment:
Why is the commented out? 
Whenever you comment out a piece of code, write a short message about why it has been commented out.